### DISCUSSION TITLE
New Subclipse update sites

### DISCUSSION THREAD

**Discussion by markphip** [subclipse/subclipse#157] [General]
I received notification from Bintray.com that they are shutting down as of May 2021. We use Bintray to host our Eclipse update sites for Subclipse. Due to this change, I have decided to move the update sites to GitHub and serve them using GitHub pages. I have updated the Subclipse wiki and the Eclipse Marketplace listing for Subclipse to point to the new URL's and I have updated the GitHub Actions we use to build and deploy Subclipse to deploy to the new sites.

These are the new URL's to use:

Latest: https://subclipse.github.io/updates/
4.3.x:  https://subclipse.github.io/updates/subclipse/4.3.x
Snapshots:  https://subclipse.github.io/snapshots/

Reminder, these are URL's to use in the Eclipse Install/Update UI, they are not meant for web browser consumption.

**Comment by bhannold** [comment]
Can you bring the 1.9 and 1.10 versions to GitHub too?  I'm supporting old tools that want those old versions of Subclipse.  Thanks.

**Reply by markphip** [reply]
You will need to be more specific. AFAIK, everything that ever existed was brought over. The URL's are in the wiki.  https://github.com/subclipse/subclipse/wiki

There never was a Subclipse 1.9 release that I can recall. Perhaps you mean SVN 1.9? That is mentioned on the wiki and would be Subclipse 4.2.x
